package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -0,0 +1,420 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""H5MD trajectories --- :mod:`MDAnalysis.coordinates.H5MD`
+========================================================
+
+The `H5MD`_ file format is based upon `HDF5`_, which makes use of parallel
+file system features through the MPI-IO interface of the HDF5 library.
+The reader currently uses the `H5PY`_ library to access data from an H5MD file.
+
+.. _`H5MD`: https://nongnu.org/h5md/index.html
+.. _`HDF5`: https://www.hdfgroup.org/solutions/hdf5/
+.. _`H5PY`: http://docs.h5py.org/en/stable/
+
+
+Classes
+-------
+
+.. autoclass:: Timestep
+   :members:
+
+   .. attribute:: positions
+
+      coordinates of the atoms as a :class:`numpy.ndarray` of shape
+      `(n_atoms, 3)`
+
+   .. attribute:: velocities
+
+      velocities of the atoms as a :class:`numpy.ndarray` of shape
+      `(n_atoms, 3)`; only available if the trajectory contains velocities
+      or if the *velocities* = ``True`` keyword has been supplied.
```

Comment:
```suggestion
or if the `velocities` = ``True`` keyword has been supplied.
```
We use single backtics for arguments.